vendor/github.com/moby/buildkit/frontend/dockerfile/builder/build.go

```diff
@@ -308,7 +308,7 @@ func Build(ctx context.Context, c client.Client) (*client.Result, error) {
 	for i, tp := range targetPlatforms {
 		func(i int, tp *specs.Platform) {
 			eg.Go(func() error {
-				st, img, err := dockerfile2llb.Dockerfile2LLB(ctx, dtDockerfile, dockerfile2llb.ConvertOpt{
+				st, img, baseImg, err := dockerfile2llb.Dockerfile2LLB(ctx, dtDockerfile, dockerfile2llb.ConvertOpt{
```

Comment:
FWIW; the changes made in the `vendor` directory should be upstreamed too github.com/moby/buildkit.

I'll add a `needs-vendoring` label